Sources/Guaka/Flag/Flag+Utilities.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  Flag+Utilities.swift
+//  Guaka
+//
+//  Created by Omar Abdelhafith on 12/11/2016.
+//
+//
+
+import Foundation
+
+
+extension Flag {
```

Comment:
Moved part of the code from Flag.swift to its own extension file (just for improving readability)